### PR TITLE
Make changes to the request form for Jennifer for Summer 2021 reopen.

### PR DESCRIPTION
**Pull request by markpbaggett**
**JIRA Ticket**: [AS-126](https://jirautk.atlassian.net/browse/AS-126)

# What does this Pull Request do?

Makes changes to the request form based on the request from Jennifer Beals on July 6, 2021.

# What's new?
2 values in the English locale form have been made to reflect that it requires 2 days minimum to pull materials.

# How should this be tested?

Pull this in your favorite dev environment.  Is the change formatted correctly? Should the change be made elsewhere? The locale change is based on previous changes.

# Interested parties
@CanOfBees @mathewjordan

### CODE REVIEW

File: public/locales/en.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ en:
     date: Anticipated arrival date
     note: Please include information listed below
     note_additional: |
-      <li><strong>Date of Expected Use:</strong> Special Collections is open by appointment only.  You must schedule a time to view the requested materials through the registration form on the <a href="https://www.lib.utk.edu/special">Special Collections home page</a>.</li>
+      <li><strong>Date of Expected Use:</strong> Collections are stored offsite.  A minimum of 2 full business days are needed to retrieve these items for use.
```
Comment (mathewjordan) [inline public/locales/en.yml:76]:
```suggestion
      <li><strong>Date of Expected Use:</strong> Collections are stored offsite.  A minimum of 2 full business days are needed to retrieve these items for use.</li>
```

Comment (mathewjordan) [reply public/locales/en.yml:76]: @markpbaggett This looks good other than the need to close the list item.

**Review by mathewjordan — CHANGES_REQUESTED** [review]

File: public/locales/en.yml (hunk 1 above)
Comment (markpbaggett) [reply public/locales/en.yml:76]: @mathewjordan 🤦🏻

**Review by CanOfBees — APPROVED** [review]
Thanks for catching that missing `</li>`. This looks good to me.

**Review by mathewjordan — APPROVED** [review]
:shipit:

**Pull request merged**